src/material-experimental/mdc-tabs/_tabs-common.scss

```diff
@@ -1,12 +1,28 @@
 @use '@material/ripple' as mdc-ripple;
 @use '@material/tab' as mdc-tab;
+@use '@material/tab-indicator' as mdc-tab-indicator;
 @use 'sass:map';
 @use '../../material/core/style/vendor-prefixes';
 @use '../../cdk/a11y';
 @use '../mdc-helpers/mdc-helpers';
 
 $mat-tab-animation-duration: 500ms !default;
 
+// Combines the various structural styles we need for the tab group and tab nav bar.
+@mixin structural-styles {
+  @include mdc-tab.without-ripple($query: mdc-helpers.$mat-base-styles-query);
+  @include mdc-tab-indicator.core-styles($query: mdc-helpers.$mat-base-styles-query);
+
```

Comment:
This mixin is somewhat small at the moment, but we'll need to add the `theme-styles` mixins to it once MDC fixes them.